### DISCUSSION TITLE
Change for sentries

### DISCUSSION THREAD

**Discussion by element109** [ReactiveDrop/reactivedrop_public_src#192] [Ideas]
Require tech class to equip sentries in the lobby, but any class can carry and activate them in game.

**Comment by element109** [comment]
This would mean that each tech could take only one sentry from the lobby. Cheat convars would be created to disable the limit but would be enabled by default.

**Comment by mithrand0** [comment]
Is there a balancing issue right now with sentries?

**Reply by wawawawawawawa** [reply]
On hard diff, it's mostly used to block spawn using stairs, and indeed it can even create lot of lags on like rng4 if you block near some infinite/very big spawner.
On casual diff, I guess it makes ppl just hide at start, and melee? That could be silly but work fine on like vegas, in low diff and/or fist offhand
Not entirely sure there

**Comment by mithrand0** [comment]
This topic is quite old, and no balance changes are planned for sentries right now. Closing it.
